src/expandableCalendar/index.tsx

```diff
@@ -174,12 +174,17 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
     return CLOSED_HEIGHT + (WEEK_HEIGHT * (numberOfWeeks.current - 1)) + (hideKnob ? 12 : KNOB_CONTAINER_HEIGHT) + (constants.isAndroid ? 3 : 0);
   };
   const openHeight = useRef(getOpenHeight());
-  const closedHeight = useRef(CLOSED_HEIGHT + (hideKnob || Number(numberOfDays) > 1 ? 0 : KNOB_CONTAINER_HEIGHT));
+  const closedHeight = useMemo(() => CLOSED_HEIGHT + (hideKnob || Number(numberOfDays) > 1 ? 0 : KNOB_CONTAINER_HEIGHT), [numberOfDays]);
 
-  const startHeight = isOpen ? openHeight.current : closedHeight.current;
+  const startHeight = useMemo(() => isOpen ? openHeight.current : closedHeight, [closedHeight]);
```

Comment:
missing dep 'isOpen'

Reply:
Actualy I'm not sure memoizing this is a good idea as it effects other non-memoized consts

Reply:
it affects memoized consts or is used as an initializer for useRef references, also added the missing dep